.travis.yml

```diff
@@ -14,8 +14,9 @@ install:
   - sudo apt-get install docker-engine
   - docker swarm init --advertise-addr 127.0.0.1
   - make install
```

Comment:
Isn't it possible to get rid of the `make install` for travis, now that everything is containerized ?